mkpkg

```diff
@@ -165,9 +165,7 @@ class Functions:
                     for b in self._expand_list(bins)])
 
     def mapbins(self, src, dst, *bins):
-        return sorted(['{src}/{bin}={dst}/{bin}{suffix}'.format(
-                src=src, dst=dst, bin=b, suffix=self.pkg.vars['suffix'])
-                    for b in self._expand_list(bins)])
+        return self.mapfiles(src, dst, *bins, append_suffix = True)
```

Comment:
Not a biggie, but in Python the convention is to not leave spaces around the `=` when passing keyword arguments.